diff -ur a/src/common/engine/m_joy.cpp b/src/common/engine/m_joy.cpp
--- a/src/common/engine/m_joy.cpp	Source
+++ b/src/common/engine/m_joy.cpp	Modified
@@ -59,7 +59,7 @@
 
 // PUBLIC DATA DEFINITIONS -------------------------------------------------
 
-CUSTOM_CVARD(Bool, use_joystick, false, CVAR_ARCHIVE|CVAR_GLOBALCONFIG|CVAR_NOINITCALL, "enables input from the joystick if it is present") 
+CUSTOM_CVARD(Bool, use_joystick, true, CVAR_ARCHIVE|CVAR_GLOBALCONFIG|CVAR_NOINITCALL, "enables input from the joystick if it is present") 
 {
 #ifdef _WIN32
 	joy_ps2raw->Callback();
diff -ur a/src/common/platform/posix/sdl/i_input.cpp b/src/common/platform/posix/sdl/i_input.cpp
--- a/src/common/platform/posix/sdl/i_input.cpp	Source
+++ b/src/common/platform/posix/sdl/i_input.cpp	Modified
@@ -180,12 +180,10 @@
 {
 	// Clear out any mouse movement.
 	SDL_GetRelativeMouseState (NULL, NULL);
-	SDL_SetRelativeMouseMode (SDL_TRUE);
 }
 
 void I_ReleaseMouseCapture()
 {
-	SDL_SetRelativeMouseMode (SDL_FALSE);
 }
 
 static void MouseRead ()
