diff -ur a/src/common/console/c_console.cpp b/src/common/console/c_console.cpp
--- a/src/common/console/c_console.cpp	Source
+++ b/src/common/console/c_console.cpp	Modified
@@ -134,7 +134,7 @@
 static char *work = NULL;
 static int worklen = 0;
 
-CUSTOM_CVAR(Int, con_scale, 0, CVAR_ARCHIVE)
+CUSTOM_CVAR(Int, con_scale, 4, CVAR_ARCHIVE)
 {
 	if (self < 0) self = 0;
 }
diff -ur a/src/common/rendering/v_video.cpp b/src/common/rendering/v_video.cpp
--- a/src/common/rendering/v_video.cpp	Source
+++ b/src/common/rendering/v_video.cpp	Modified
@@ -142,7 +142,7 @@
 }
 
 
-CUSTOM_CVAR(Int, uiscale, 0, CVAR_ARCHIVE | CVAR_NOINITCALL)
+CUSTOM_CVAR(Int, uiscale, 3, CVAR_ARCHIVE | CVAR_NOINITCALL)
 {
 	if (self < 0)
 	{
diff -ur a/src/common/statusbar/base_sbar.cpp b/src/common/statusbar/base_sbar.cpp
--- a/src/common/statusbar/base_sbar.cpp	Source
+++ b/src/common/statusbar/base_sbar.cpp	Modified
@@ -57,9 +57,9 @@
 IMPLEMENT_CLASS(DHUDFont, false, false);
 
 
-CVAR(Color, crosshaircolor, 0xff0000, CVAR_ARCHIVE);
-CVAR(Int, crosshairhealth, 2, CVAR_ARCHIVE);
-CVARD(Float, crosshairscale, 0.5, CVAR_ARCHIVE, "changes the size of the crosshair");
+CVAR(Color, crosshaircolor, 0x00ff1e, CVAR_ARCHIVE);
+CVAR(Int, crosshairhealth, 1, CVAR_ARCHIVE);
+CVARD(Float, crosshairscale, 0.25, CVAR_ARCHIVE, "changes the size of the crosshair");
 CVAR(Bool, crosshairgrow, false, CVAR_ARCHIVE);
 
 CUSTOM_CVARD(Float, hud_scalefactor, 1.f, CVAR_ARCHIVE, "changes the hud scale")
diff -ur a/src/console/c_notifybuffer.cpp b/src/console/c_notifybuffer.cpp
--- a/src/console/c_notifybuffer.cpp	Source
+++ b/src/console/c_notifybuffer.cpp	Modified
@@ -59,7 +59,7 @@
 CVAR(Bool, con_centernotify, false, CVAR_ARCHIVE)
 CVAR(Bool, con_pulsetext, false, CVAR_ARCHIVE)
 
-CUSTOM_CVAR(Int, con_scaletext, 0, CVAR_ARCHIVE)		// Scale notify text at high resolutions?
+CUSTOM_CVAR(Int, con_scaletext, 4, CVAR_ARCHIVE)		// Scale notify text at high resolutions?
 {
 	if (self < 0) self = 0;
 }
diff -ur a/src/g_game.cpp b/src/g_game.cpp
--- a/src/g_game.cpp	Source
+++ b/src/g_game.cpp	Modified
@@ -205,7 +205,7 @@
 
 #define SLOWTURNTICS	6 
 
-CVAR (Bool,		cl_run,			false,	CVAR_GLOBALCONFIG|CVAR_ARCHIVE)		// Always run?
+CVAR (Bool,		cl_run,			true,	CVAR_GLOBALCONFIG|CVAR_ARCHIVE)		// Always running from evil?
 CVAR (Bool,		freelook,		true,	CVAR_GLOBALCONFIG|CVAR_ARCHIVE)		// Always mlook?
 CVAR (Bool,		lookstrafe,		false,	CVAR_GLOBALCONFIG|CVAR_ARCHIVE)		// Always strafe with mouse?
 CVAR (Float,	m_forward,		1.f,	CVAR_GLOBALCONFIG|CVAR_ARCHIVE)
@@ -2262,7 +2262,7 @@
 
 CVAR (Int, autosavenum, 0, CVAR_NOSET|CVAR_ARCHIVE|CVAR_GLOBALCONFIG)
 static int nextautosave = -1;
-CVAR (Int, disableautosave, 0, CVAR_ARCHIVE|CVAR_GLOBALCONFIG)
+CVAR (Int, disableautosave, 2, CVAR_ARCHIVE|CVAR_GLOBALCONFIG)
 CVAR (Bool, saveloadconfirmation, true, CVAR_ARCHIVE | CVAR_GLOBALCONFIG) // [mxd]
 CUSTOM_CVAR (Int, autosavecount, 4, CVAR_ARCHIVE|CVAR_GLOBALCONFIG)
 {
diff -ur a/src/g_statusbar/shared_sbar.cpp b/src/g_statusbar/shared_sbar.cpp
--- a/src/g_statusbar/shared_sbar.cpp	Source
+++ b/src/g_statusbar/shared_sbar.cpp	Modified
@@ -89,7 +89,7 @@
 EXTERN_CVAR(Bool, inter_subtitles)
 EXTERN_CVAR(Bool, ui_screenborder_classic_scaling)
 
-CVAR(Int, hud_scale, -1, CVAR_ARCHIVE);
+CVAR(Int, hud_scale, 0, CVAR_ARCHIVE);
 CVAR(Bool, log_vgafont, false, CVAR_ARCHIVE)
 CVAR(Bool, hud_oldscale, true, CVAR_ARCHIVE)
 
@@ -105,7 +105,7 @@
 
 
 // Stretch status bar to full screen width?
-CUSTOM_CVAR (Int, st_scale, -1, CVAR_ARCHIVE)
+CUSTOM_CVAR (Int, st_scale, 2, CVAR_ARCHIVE)
 {
 	if (self < -1)
 	{
@@ -123,8 +123,8 @@
 EXTERN_CVAR(Bool, hud_aspectscale)
 
 CVAR (Bool, crosshairon, true, CVAR_ARCHIVE);
-CVAR (Int, crosshair, 0, CVAR_ARCHIVE)
-CVAR (Bool, crosshairforce, false, CVAR_ARCHIVE)
+CVAR (Int, crosshair, 2, CVAR_ARCHIVE)
+CVAR (Bool, crosshairforce, true, CVAR_ARCHIVE)
 CUSTOM_CVAR(Int, am_showmaplabel, 2, CVAR_ARCHIVE)
 {
 	if (self < 0 || self > 2) self = 2;
