diff -ur a/src/g_cvars.cpp b/src/g_cvars.cpp
--- a/src/g_cvars.cpp	2024-06-13 19:37:29.000000000 -0400
+++ b/src/g_cvars.cpp	2024-06-23 12:07:54.247246400 -0400
@@ -122,10 +122,10 @@
 
 // [BC] Allow the maximum number of particles to be specified by a cvar (so people
 // with lots of nice hardware can have lots of particles!).
-CUSTOM_CVAR(Int, r_maxparticles, 4000, CVAR_ARCHIVE | CVAR_NOINITCALL)
+CUSTOM_CVAR(Int, r_maxparticles, 2500, CVAR_ARCHIVE | CVAR_NOINITCALL)
 {
 	if (self == 0)
-		self = 4000;
+		self = 2500;
 	else if (self > 65535)
 		self = 65535;
 	else if (self < 100)
diff -ur a/src/playsim/p_effect.cpp b/src/playsim/p_effect.cpp
--- a/src/playsim/p_effect.cpp	2024-06-13 19:37:29.000000000 -0400
+++ b/src/playsim/p_effect.cpp	2024-06-13 20:12:03.396819000 -0400
@@ -57,7 +57,7 @@
 #pragma warning(disable: 6011) // dereference null pointer in thinker iterator
 #endif
 
-CVAR (Int, cl_rockettrails, 1, CVAR_ARCHIVE);
+CVAR (Int, cl_rockettrails, 0, CVAR_ARCHIVE);
 CVAR (Bool, r_rail_smartspiral, false, CVAR_ARCHIVE);
 CVAR (Int, r_rail_spiralsparsity, 1, CVAR_ARCHIVE);
 CVAR (Int, r_rail_trailsparsity, 1, CVAR_ARCHIVE);
