diff -ur a/src/am_map.cpp b/src/am_map.cpp
--- a/src/am_map.cpp	2025-06-15 20:12:44.689600019 -0400
+++ b/src/am_map.cpp	2025-06-15 20:22:19.474038903 -0400
@@ -171,7 +171,7 @@
 CVAR(Bool, am_showtime, true, CVAR_ARCHIVE);
 CVAR(Bool, am_showtotaltime, false, CVAR_ARCHIVE);
 CVAR(Bool, am_showlevelname, true, CVAR_ARCHIVE);
-CVAR(Int, am_colorset, 0, CVAR_ARCHIVE);
+CVAR(Int, am_colorset, 1, CVAR_ARCHIVE);
 CVAR(Bool, am_customcolors, true, CVAR_ARCHIVE);
 CVAR(Int, am_map_secrets, 1, CVAR_ARCHIVE);
 CVAR(Int, am_drawmapback, 1, CVAR_ARCHIVE);
diff -ur a/src/common/console/c_console.cpp b/src/common/console/c_console.cpp
--- a/src/common/console/c_console.cpp	2025-06-15 20:12:44.693600019 -0400
+++ b/src/common/console/c_console.cpp	2025-06-15 21:20:21.029596578 -0400
@@ -128,7 +128,7 @@
 static char *work = NULL;
 static int worklen = 0;
 
-CUSTOM_CVAR(Int, con_scale, 0, CVAR_ARCHIVE)
+CUSTOM_CVAR(Int, con_scale, 4, CVAR_ARCHIVE)
 {
 	if (self < 0) self = 0;
 }
diff -ur a/src/common/rendering/v_video.cpp b/src/common/rendering/v_video.cpp
--- a/src/common/rendering/v_video.cpp	2025-06-15 20:12:44.721600013 -0400
+++ b/src/common/rendering/v_video.cpp	2025-06-15 20:22:18.714038330 -0400
@@ -130,7 +130,7 @@
 }
 
 
-CUSTOM_CVAR(Int, uiscale, 0, CVAR_ARCHIVE | CVAR_NOINITCALL)
+CUSTOM_CVAR(Int, uiscale, 2, CVAR_ARCHIVE | CVAR_NOINITCALL)
 {
 	if (self < 0)
 	{
diff -ur a/src/common/statusbar/base_sbar.cpp b/src/common/statusbar/base_sbar.cpp
--- a/src/common/statusbar/base_sbar.cpp	2025-06-15 20:12:44.733600011 -0400
+++ b/src/common/statusbar/base_sbar.cpp	2025-06-15 20:22:18.018037807 -0400
@@ -57,9 +57,9 @@
 IMPLEMENT_CLASS(DHUDFont, false, false);
 
 
-CVAR(Color, crosshaircolor, 0xff0000, CVAR_ARCHIVE);
-CVAR(Int, crosshairhealth, 2, CVAR_ARCHIVE);
-CVARD(Float, crosshairscale, 0.5, CVAR_ARCHIVE, "changes the size of the crosshair");
+CVAR(Color, crosshaircolor, 0x00ff1e, CVAR_ARCHIVE);
+CVAR(Int, crosshairhealth, 1, CVAR_ARCHIVE);
+CVARD(Float, crosshairscale, 0.25, CVAR_ARCHIVE, "changes the size of the crosshair");
 CVAR(Bool, crosshairgrow, false, CVAR_ARCHIVE);
 
 CUSTOM_CVARD(Float, hud_scalefactor, 1.f, CVAR_ARCHIVE, "changes the hud scale")
diff -ur a/src/console/c_notifybuffer.cpp b/src/console/c_notifybuffer.cpp
--- a/src/console/c_notifybuffer.cpp	2025-06-15 20:12:44.765600005 -0400
+++ b/src/console/c_notifybuffer.cpp	2025-06-15 21:20:19.109596839 -0400
@@ -59,7 +59,7 @@
 CVAR(Bool, con_centernotify, false, CVAR_ARCHIVE)
 CVAR(Bool, con_pulsetext, false, CVAR_ARCHIVE)
 
-CUSTOM_CVAR(Int, con_scaletext, 0, CVAR_ARCHIVE)		// Scale notify text at high resolutions?
+CUSTOM_CVAR(Int, con_scaletext, 4, CVAR_ARCHIVE)		// Scale notify text at high resolutions?
 {
 	if (self < 0) self = 0;
 }
diff -ur a/src/g_game.cpp b/src/g_game.cpp
--- a/src/g_game.cpp	2025-06-15 20:12:44.769600004 -0400
+++ b/src/g_game.cpp	2025-06-15 20:22:21.138040149 -0400
@@ -2235,7 +2235,7 @@
 
 CVAR (Int, autosavenum, 0, CVAR_NOSET|CVAR_ARCHIVE|CVAR_GLOBALCONFIG)
 static int nextautosave = -1;
-CVAR (Int, disableautosave, 0, CVAR_ARCHIVE|CVAR_GLOBALCONFIG)
+CVAR (Int, disableautosave, 2, CVAR_ARCHIVE|CVAR_GLOBALCONFIG)
 CVAR (Bool, saveloadconfirmation, true, CVAR_ARCHIVE | CVAR_GLOBALCONFIG) // [mxd]
 CUSTOM_CVAR (Int, autosavecount, 4, CVAR_ARCHIVE|CVAR_GLOBALCONFIG)
 {
diff -ur a/src/g_statusbar/shared_sbar.cpp b/src/g_statusbar/shared_sbar.cpp
--- a/src/g_statusbar/shared_sbar.cpp	2025-06-15 20:12:44.769600004 -0400
+++ b/src/g_statusbar/shared_sbar.cpp	2025-06-15 20:22:57.050066218 -0400
@@ -105,7 +105,7 @@
 
 
 // Stretch status bar to full screen width?
-CUSTOM_CVAR (Int, st_scale, 0, CVAR_ARCHIVE)
+CUSTOM_CVAR (Int, st_scale, 2, CVAR_ARCHIVE)
 {
 	if (self < -1)
 	{
@@ -123,8 +123,8 @@
 EXTERN_CVAR(Bool, hud_aspectscale)
 
 CVAR (Bool, crosshairon, true, CVAR_ARCHIVE);
-CVAR (Int, crosshair, 0, CVAR_ARCHIVE)
-CVAR (Bool, crosshairforce, false, CVAR_ARCHIVE)
+CVAR (Int, crosshair, 2, CVAR_ARCHIVE)
+CVAR (Bool, crosshairforce, true, CVAR_ARCHIVE)
 CUSTOM_CVAR(Int, am_showmaplabel, 2, CVAR_ARCHIVE)
 {
 	if (self < 0 || self > 2) self = 2;
diff -ur a/src/menu/doommenu.cpp b/src/menu/doommenu.cpp
--- a/src/menu/doommenu.cpp	2025-06-15 20:12:44.777600002 -0400
+++ b/src/menu/doommenu.cpp	2025-06-15 20:23:39.538095059 -0400
@@ -76,7 +76,7 @@
 EXTERN_CVAR(Bool, show_messages)
 EXTERN_CVAR(Float, hud_scalefactor)
 
-CVAR(Bool, m_simpleoptions, true, CVAR_ARCHIVE|CVAR_GLOBALCONFIG)
+CVAR(Bool, m_simpleoptions, false, CVAR_ARCHIVE|CVAR_GLOBALCONFIG)
 
 typedef void(*hfunc)();
 DMenu* CreateMessageBoxMenu(DMenu* parent, const char* message, int messagemode, bool playsound, FName action = NAME_None, hfunc handler = nullptr);
